tpdatasrc/tpgamefiles/rules/d20_actions/action02900_abjurant_champion_arcane_boost.py

```diff
@@ -0,0 +1,22 @@
+from toee import *
+import tpactions
+
+def GetActionName():
+	return "Arcane Boost"
+
+def GetActionDefinitionFlags():
+	return D20ADF_None
+	
+def GetTargetingClassification():
+	return D20TC_CastSpell
```

Comment:
So why do you actually need this to be d20tc_castspell?

Reply:
I need the spell packet for getting the spell level and debiting the spell.  If I do D20TC_Target0 (which would not show targeting), I don't get a spell packet.

Reply:
Well that data is set according to what's already stored in evt_obj.d20a.spell_data, so you could just retrace it by creating a tpdp.SpellPacket(), setting its fields just like D20::TargetCheck() does and then calling debit from it. If there isn't already you should add a helper method for creating a SpellPacket from D20SpellData.

Reply:
Thanks!  That is an interesting idea.  Looks like a spell packet can currently only be created from a spell id but I'll see about making a way to construct one from D20SpellData.

Reply:
That seemed to work great.  I pushed up the change.  I'll test just a bit more and remove the draft tag on Friday.